### DISCUSSION TITLE
Migration from DockerHub to GitHub Container Registry

### DISCUSSION THREAD

**Discussion by NebN** [NebN/unraid-simple-monitoring-api#8] [Announcements]
**I have migrated from DockerHub to GitHub Container Registry.**

If you have installed before April 2024 please reinstall or manually change  
`Repository` to  
`ghcr.io/nebn/unraid-simple-monitoring-api:latest`  
![image](https://github.com/NebN/unraid-simple-monitoring-api/assets/57036949/3a2d8617-ee61-4eac-a76e-e1076408152b)

And optionally
`Registry URL` to  
`https://github.com/NebN/unraid-simple-monitoring-api/pkgs/container/unraid-simple-monitoring-api`  
![image](https://github.com/NebN/unraid-simple-monitoring-api/assets/57036949/1e731fda-bc4e-42ab-b4f7-4617e1897d49)  

I will keep pushing to DockerHub for now, but would like to definitively migrate.

**Comment by chandz05** [comment]
Thank you! I've updated my container
